tests/dragonfly/memory_test.py

```diff
@@ -51,9 +52,10 @@ async def test_rss_used_mem_gap(df_server: DflyInstance, type, keys, val_size, e
     # It could be the case that the machine is configured to use swap if this assertion fails
     assert delta > 0
     assert delta < max_unaccounted
-    delta = info["used_memory_rss"] - info["object_used_memory"]
-    # TODO investigate why it fails on string
-    if type == "JSON" or type == "STREAM":
+
+    if type != "LIST":
```

Comment:
So we still have problem with list?

Reply:
Yes, Roman recently talked about taking ownership over the lists code. We don't track it too well, and we have a slow version of used memory. It's still WIP.

Reply:
We now got a list class if I remember correctly? It should be easier to bake in "memory tracking"